engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -604,6 +604,9 @@ namespace dmGameSystem
         gui_component->m_Scene = dmGui::NewScene(scene_resource->m_GuiContext, &scene_params);
         dmGui::HScene scene = gui_component->m_Scene;
 
+        gui_component->m_resourcePropertyPointers.SetCapacity(scene_params.m_MaxTextures + scene_params.m_MaxFonts);
```

Comment:
My point with the dmArray implementation is that it is a dynamic array.
There's really rarely any need to preallocate this data, as most users will never use this feature.

Instead, call OffsetCapacity() if the buffer is full when we want to add another item. We can still have an upper limit if we want to.

Reply:
👍makes sense, I'll fix that